src/main/java/org/traccar/protocol/TopinProtocolDecoder.java

```diff
@@ -78,6 +96,10 @@ protected Object decode(
             ByteBuf content = Unpooled.buffer();
             content.writeByte(deviceSession != null ? 0x01 : 0x44);
             sendResponse(channel, length, type, content);
+
+            //update time directly after login
+            updateTime(channel, MSG_TIME_UPDATE);
```

Comment:
This doesn't seem like a requirement from the protocol, so this shouldn't be here.

Reply:
It is not in the documentation, but the TOPIN default server does this to ensure that no data with incorrect timestamp is send to the server. This is the reason why this change was added! If the power to the tracker was lost the first few packages will have an incorrect timestamp (non-utc). This leads to incorrect data in the traccar system.

Reply:
OK, let's keep it, but please remove comment.

Reply:
Fixed in 1d7d66a.